### DISCUSSION TITLE
Key up/down status

### DISCUSSION THREAD

**Discussion by swalok47716** [pqrs-org/Karabiner-Elements#4027] [Q&A] [answered]
Is there a way to do key_up or down for a specific key? Or a similar things like so?

Seems like there's no such key code for these key status

Thanks for helping

**Comment by tekezo** [comment, marked as answer]
Currently, there is no way to send only key_down or key_up.
Each key_down and key_up is sent corresponding to the pressing and releasing of a key.

**Reply by swalok47716** [reply]
Got it.
